app/consumers/concerns/report_parsing.rb

```diff
@@ -96,7 +96,10 @@ def b64_identity
     end
 
     def report_contents
-      @report_contents ||= SafeDownloader.download(url)
+      @report_contents ||= SafeDownloader.download_reports(
+        url,
+        ssl_only: Settings.report_download_ssl_only
```

Comment:
Why pass this in when we can just check this global setting directly?

Reply:
Idk, it made more sense to me to do it this way and have it as an option. :slightly_smiling_face: